### ISSUE TITLE
Name can't as same as ID

### ISSUE THREAD

**Issue by ericzhong** [angular-gantt/angular-gantt#534]
{id:'xxx', name: '001'},
{id:'01', name: 'Dev', children: ['001', '002', '003', '004']},
{id:'001', name: 'Dev1'},
{id:'002', name: 'Dev2'},
{id:'003', name: 'Dev3'},
{id:'004', name: 'Dev4'}

Display:
Dev
----001
----Dev2
----Dev3
----Dev4
Dev1

**Comment by ericzhong** [comment]
Use 'name' as ID, keep Unique and use 'content' as Issue Name to display.

**Comment by ericzhong** [comment]
If use 'name' as Issue ID, "filter rows" will break, it becomes to filter IDs.
So, must fix this bug, only use 'id' for Issue ID and children/parent relationship.

**Comment by Toilal** [comment]
Yes, I think it would be safer, i wish some users could use this plugin without defining IDs in model, but seems it's not safe ...

**Comment by B3nCr** [comment]
Name is not unique among child rows and tasks in our project but ID is.
